tests/PeriodCollectionTest.php

```diff
@@ -208,6 +208,59 @@ public function reduce()
         $this->assertEquals(4, $totalLength);
     }
 
+    /**
+     * @test
+     *
+     * A           [=======] [===============]
+     *
+     * SUBTRACT                      [=]
+     *
+     * RESULT      [=======] [======]   [====]
+     */
+    public function subtract_a_period_from_period_collection()
+    {
+        $collection = new PeriodCollection(
+            Period::make('1987-02-01', '1987-02-10'),
+            Period::make('1987-02-11', '1987-02-28')
+        );
+
+        $removePeriod = Period::make('1987-02-20', '1987-02-21');
+
+        $this->assertEquals(1, $collection->overlapAll(new PeriodCollection($removePeriod))->count());
+
+        $result = $collection->subtract($removePeriod);
+
+        $this->assertEquals(0, $result->overlapAll(new PeriodCollection($removePeriod))->count());
```

Comment:
Use hard coded date assertions here instead. Tip: you can use something like `$this->assertTrue(Period::fromString('[1987-02-01,1987-02-19]')->equals($result[0])`